app/services/TwoStepVerification.scala

```diff
@@ -39,37 +38,43 @@ trait TwoStepVerification {
 
   def twoStepVerificationThrottle: TwoStepVerificationThrottle
 
-  case class Biz2SVRule(name: String, conditions: List[Condition])
+  def twoStepVerificationRuleFactory = new TwoStepVerificationUserSegmentFactory {}
 
-  private val biz2svRule = Biz2SVRule("sa", List(not(HasStrongCredentials), GGEnrolmentsAvailable, HasOnlyOneEnrolment, HasSelfAssessmentEnrolments, not(HasRegisteredFor2SV)))
+  private case class Biz2SVRule(name: String, conditions: List[Condition], adminLocations: ThrottleLocations, assistantLocations: ThrottleLocations)
 
-  val continueToAccountUrl = s"${ExternalUrls.taxAccountRouterHost}/account"
+  private val commonRules = List(not(HasRegisteredFor2SV), not(HasStrongCredentials), GGEnrolmentsAvailable)
```

Comment:
I think it's good to have this, I was just thinking of a better name as this is a list of conditions - something like `base2SVConditions`?